brave/brave-core
# Support importing from Whale browser

fix https://github.com/brave/brave-browser/issues/27641

![image](https://user-images.githubusercontent.com/6786187/211722191-9ad97583-e9df-469a-a2cb-de9a19f1d5c0.png)

![image](https://user-images.githubusercontent.com/6786187/211738037-19b9b2ac-9c57-4631-b695-7b180045321d.png)


Resolves 

## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Install Whale browser from https://whale.naver.com/download and launch Whale browser
2. Make some history, bookmarks and passwords and install some extensions
3. Start import from brave:://welcome or brave://settings/importData
4. Check importing is done propertly

**_NOTE_** : password importing is only supported on macOS
